src/mon/PGMap.cc

```diff
@@ -4559,6 +4560,9 @@ int reweight::by_utilization(
   // but aggressively adjust weights up whenever possible.
   const double underload_util = average_util;
 
+  // a measure of how uneven the data distribution is and hence how urgently reweighting is needed.
+  double reweight_urgency = 0.0;
```

Comment:
please move the definition down to https://github.com/ceph/ceph/pull/16361/commits/b576d8875e64ae57d23bc19edd2cf1edcb16cd35#diff-ecab4c883be988760d61a8a883ddc23fR4585. we can postpone the definition to where it is used for the first time.